City-of-Helsinki/drupal-hdbt
# UHF-1747 Webpack update

How to test: 
- Checkout this branch, install updated webpack tools and compile the styles/js by running `nvm use && npm i && npm run build`
- Check that the site still works. Styles are as before, icons work and js functionalities work.

Check also:
https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/51
https://github.com/City-of-Helsinki/drupal-helfi-platform/pull/38
